shared/js/project.js

```diff
@@ -464,6 +496,13 @@ function executeCallback(callback, data) {
 }
 
 /**
+ * is the current project (if any) editable by the logged in user (if any)?
+ */
+function isEditable() {
```

Comment:
That's why I set hidden as well as frozen (so it's not listed in /projects), but I agree, we should try to avoid proliferating settings/properties/etc (especially when we essentially have two consumers of this API). Should hidden be a state too?